Makefile

```diff
@@ -12,80 +15,86 @@ GREEN=`tput setaf 2`
 RESET=`tput sgr0`
 YELLOW=`tput setaf 3`
 
-all: .installed.cfg
+PLONE5=5.2.7
+PLONE6=6.0.0a4
+
+PACKAGE_NAME=kitconcept.contentcreator
+PACKAGE_PATH=src/
+CHECK_PATH=setup.py $(PACKAGE_PATH)
 
 # Add the following 'help' target to your Makefile
 # And add help text after each target name starting with '\#\#'
 .PHONY: help
 help: ## This help message
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
-.PHONY: Update Makefile and Buildout
-update: ## Update Make and Buildout
-	wget -O Makefile https://raw.githubusercontent.com/kitconcept/buildout/master/Makefile
-	wget -O requirements.txt https://raw.githubusercontent.com/kitconcept/buildout/master/requirements.txt
-	wget -O plone-4.3.x.cfg https://raw.githubusercontent.com/kitconcept/buildout/master/plone-4.3.x.cfg
-	wget -O plone-5.1.x.cfg https://raw.githubusercontent.com/kitconcept/buildout/master/plone-5.1.x.cfg
-	wget -O plone-5.2.x.cfg https://raw.githubusercontent.com/kitconcept/buildout/master/plone-5.2.x.cfg
-	wget -O travis.cfg https://raw.githubusercontent.com/kitconcept/buildout/master/travis.cfg
-	wget -O versions.cfg https://raw.githubusercontent.com/kitconcept/buildout/master/versions.cfg
-
-.installed.cfg: bin/buildout *.cfg
-	bin/buildout
-
-bin/buildout: bin/pip
-	bin/pip install --upgrade pip
-	bin/pip install -r requirements.txt
-	@touch -c $@
-
-bin/python bin/pip:
-	python$(version) -m venv . || virtualenv --clear --python=python$(version) .
-
-
-.PHONY: Build Plone 5.2
-build: .installed.cfg  ## Build Plone 5.2
-	bin/pip install --upgrade pip
-	bin/pip install -r requirements.txt
-	bin/buildout
-
-.PHONY: Build Plone 6.0
-build-plone-6.0:  bin/pip ## Build Plone 6.0
-	bin/pip install --upgrade pip
-	bin/pip install -r https://dist.plone.org/release/6.0.0a2/requirements.txt
-	bin/buildout -c plone-6.0.x.cfg
-
-.PHONY: Test
-test:  ## Test
-	bin/test
-
-.PHONY: Test Performance
-test-performance:
-	jmeter -n -t performance.jmx -l jmeter.jtl
-
-.PHONY: Code Analysis
-code-analysis:  ## Code Analysis
-	bin/code-analysis
-	if [ -f "bin/black" ]; then bin/black src/ --check ; fi
-
-.PHONY: Black
-black:  ## Black
-	bin/code-analysis
-	if [ -f "bin/black" ]; then bin/black src/ ; fi
-
-.PHONY: Build Docs
-docs:  ## Build Docs
-	bin/sphinxbuilder
-
-.PHONY: Test Release
-test-release:  ## Run Pyroma and Check Manifest
-	bin/pyroma -n 10 -d .
-
-.PHONY: Release
-release:  ## Release
-	bin/fullrelease
-
-.PHONY: Clean
-clean:  ## Clean
-	git clean -Xdf
-
-.PHONY: all clean
+bin/pip:
+	@echo "$(GREEN)==> Setup Virtual Env$(RESET)"
+	python3 -m venv .
+	bin/pip install -U pip wheel
+
+bin/black:
+	bin/pip install black
+
+bin/isort:
+	bin/pip install isort
+
+bin/flakeheaven:
+	bin/pip install flakeheaven
```

Comment:
I have not heard of flakeheaven yet. Seems interesting.

Reply:
It is. I like the way it simplify our configuration